src/erlcloud_s3.erl

```diff
@@ -1241,3 +1241,11 @@ s3_follow_redirect({error, {http_error, StatusCode, StatusLine, ErrBody, ErrHead
 %%              "test.bucket") -> "s3.eu-central-1.amazonaws.com"
 s3_endpoint_from_hostname(HostName, Bucket) ->
     HostName -- lists:flatten([Bucket, $.]).
+
+s3_endpoint_for_region(RegionName) ->
+    case RegionName of
+        "us-east-1" ->
+            "s3.amazonaws.com";
```

Comment:
i do re-call there was some, can't find it at the moment so `s3-external-1` routing is faster then just plain `s3.`